### DISCUSSION TITLE
How to create a keybind on to the key delete and/or use the esc action

### DISCUSSION THREAD

**Discussion by canTuncPekkan** [ghostty-org/ghostty#4061] [Q&A]
I want to bind the following keys cmd + delete to 

clearbol EL1          Clear line from cursor left            ^[[1K

which will clear a line in the terminal

I have tried doing this myself and am stuck on the following steps

1. setting the key bind to delete
2. running the escape sequence 

if any of you can do it, I would be grateful.

**Comment by vict0rsch** [comment]
Same here, I'm stuck around variations of

```
keybind = super+backspace=csi:1K
```

**Comment by vict0rsch** [comment]
Found a workaround:

in `~/.zshrc`:

```
bindkey '^[backward-kill-line' backward-kill-line
```

in Ghostty's config:

```
keybind = super+backspace=esc:backward-kill-line
```
